libvast/test/option_map.cpp

```diff
@@ -0,0 +1,127 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#define SUITE option_map
+#include "test.hpp"
+
+#include <limits>
+
+#include "vast/option_map.hpp"
+#include "vast/option_declaration_set.hpp"
+
+using namespace vast;
+
+namespace {
+
+struct fixture {
+  option_declaration_set decl;
+  option_map opts;
+};
+
+} // namespace <anonymous>
+
+FIXTURE_SCOPE(command_tests, fixture)
+
+TEST(option map handling) {
```

Comment:
"handling" is a very generic term that doesn't explain what's getting tested here.